source/features/softwrap-code.css

```diff
@@ -0,0 +1,14 @@
+/* For soft-wrapped code inside tables */
+.rgh-softwrapped-code {
+	table-layout: fixed;
+}
+.rgh-softwrapped-code tr:not(.js-expandable-line) .blob-num {
+	width: 50px;
+}
+.rgh-softwrapped-code tr:not(.js-expandable-line) .blob-code-inner {
```

Comment:
Only this seems to be needed (in PR Files). Either drop the rest of the code or add comments to explain it